eng/Versions.props

```diff
@@ -4,10 +4,32 @@
     <VersionPrefix>4.8.0</VersionPrefix>
     <PreReleaseVersionLabel>prerelease</PreReleaseVersionLabel>
     <AssemblyVersion>4.0.0.0</AssemblyVersion>
-    <SystemSecurityVersion>4.5.0</SystemSecurityVersion>
-    <SystemReflectionEmitVersion>4.3.0</SystemReflectionEmitVersion>
```

Comment:
Looks like you removed this property entirely.  You need to add it back with a subscription.

```xml
    <SystemReflectionEmitVersion>4.6.0-preview.19060.1</SystemReflectionEmitVersion>
```

Reply:
I changed it to `SystemReflectionEmitPackageVersion` so that it will be properly updated by Maestro. Looks like the build failed because I forgot to update a few other files

Reply:
Also Steven, if you don't want to change the property names in your build, you can just make a property in Versions.props that matches the darc dependency name and just have your existing property wrap it. Like <MyProp>$PropFromDarc</MyProp>.

We did this for the winforms package in DotNet-Trusted to avoid breaking other stuff.